includes/admin/class.llms.admin.setup.wizard.php

```diff
@@ -382,46 +125,32 @@ public function output_before_importable_course( $course ) {
 	 *
 	 * @since 4.8.0
 	 *
-	 * @param array $course Importable course data array.
 	 * @return void
 	 */
-	public function output_after_importable_course( $course ) {
+	public function output_after_importable_course(): void {
```

Comment:
```suggestion
	public function output_after_importable_course( array $course ): void {
```
Would keep this too, can you?
No real reason to change a signature, right?